GitHub-Actions/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.github/workflows/pipeline.yaml

```diff
@@ -57,7 +60,11 @@ jobs:
       - uses: actions/setup-python@v2
       - uses: aws-actions/setup-sam@v1
 
-      - name: Assume the testing pipeline user role
+      - name: Assume the testing pipeline user role via IAM user
+        if: env.PERMISSIONS_PROVIDER == 'IAM'
+        env:
+          PIPELINE_USER_ACCESS_KEY_ID: ${{ "{{" }} secrets.{{cookiecutter.pipeline_user_aws_access_key_id_variable_name}} {{ "}}" }}
+          PIPELINE_USER_SECRET_ACCESS_KEY: ${{ "{{" }} secrets.{{cookiecutter.pipeline_user_aws_secret_access_key_variable_name}} {{ "}}" }}
```

Comment:
I'm wondering if there's a way to reuse env variables to save some duplicate code here, like putting all jobs into a single job. Not sure if that would work though

Reply:
Ill move the Pipeline_User variables back to the top of the scope and just make them conditional with Jinja, and then for the the assume role steps I can do the same for the pipeline user options that get passed